src/zui/ZUIDateRangePicker/ZUIDateRangePicker/index.tsx

```diff
@@ -11,26 +11,67 @@ import {
   TextField,
   Typography,
 } from '@mui/material';
-import { CalendarToday, Clear } from '@mui/icons-material';
+import { Clear, Schedule, VisibilityOutlined } from '@mui/icons-material';
 import { DateRange, StaticDateRangePicker } from '@mui/x-date-pickers-pro';
 import dayjs, { Dayjs } from 'dayjs';
 import React, { FC, MouseEvent, useEffect, useState } from 'react';
 
+import { EyeClosed } from 'zui/icons/EyeClosed';
 import messageIds from 'zui/l10n/messageIds';
 import { useMessages, UseMessagesMap } from 'core/i18n';
 
 const rangeStr = (
   messages: UseMessagesMap<typeof messageIds.dateRange>,
   values: DateRange<Dayjs>
-): string => {
+): { icon: JSX.Element; message: string } => {
```

Comment:
Maybe rename this function, now that it no longer returns just a string?

Reply:
Ofcourse!